admin/acceptance.py

```diff
@@ -163,12 +163,14 @@ class VagrantRunner(object):
     # rather than assuming it is available.
     # https://clusterhq.atlassian.net/browse/FLOC-1163
 
-    NODE_ADDRESSES = ["172.16.255.240", "172.16.255.241"]
+    NODE_ADDRESSES = ["172.16.255.250", "172.16.255.251"]
```

Comment:
It does mean that. Given that in the event of a conflict, we would have no way of knowing what IP addresses a user was able to make use of, the only thing we can do is point them to the full CLI and node installation instructions - they would need to generate a new set of certificates and keys, copy them on to the nodes and start the services manually.